types.d.ts

```diff
@@ -1,24 +1,41 @@
 import { bits } from 'discord-rose/dist/utils/Permissions'
 import { CommandContext } from "discord-rose/dist/structures/CommandContext";
```

Comment:
`discord-rose/dist/structures/CommandContext` should be replaced with `discord-rose/dist/typings/lib`